packages/@react-spectrum/progress/docs/ProgressCircle.mdx

```diff
@@ -0,0 +1,103 @@
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/progress';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ProgressCircle} from '@react-spectrum/progress';
+import {View} from '@react-spectrum/view';
+```
+
+# ProgressCircle
+
+<p>{docs.exports.ProgressCircle.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ProgressCircle']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/circle-loader/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ProgressCircle aria-label="Loading…" value={50} />
+```
+
+## Value
+
+By default Progress Circles are controlled with the `value` prop representing current percentage as an integer.
```

Comment:
I noticed that we have `Progress Circles` as two words instead of one `ProgressCircles` throughout the docs.